guides/source/upgrading_ruby_on_rails.md

```diff
@@ -52,6 +52,15 @@ Don't forget to review the difference, to see if there were any unexpected chang
 
 Upgrading from Rails 4.2 to Rails 5.0
 -------------------------------------
+### Complex expressions in `ActiveRecord::Calculations#calculation`
+
+In Rails 4.2 the aggregate methods in `ActiveRecord::Calculations#calculation` type cast the result
```

Comment:
can we link `ActiveRecord::Calculations` to the API docs? Also it would be good to include a user facing example as well, like `sum` or `average`.